### DISCUSSION TITLE
How to handle multiple gene_reaction_rules

### DISCUSSION THREAD

**Discussion by mephenor** [SBRG/ModelPolisher#112] [Release 2.1]
While trying to fix the GPR parsing code I realized that I unilaterally added code to merge multiple gene_reaction_rules into one new term under a logical or quite some time ago.
While this only affects one or two gene reaction rules across all of bigg_models_data, if I remember correctly, it should still be clarified if this is indeed the correct handling of the situation or something different should be done here.

**Comment by Schmoho** [comment]
As far as I can tell. there seems to be no actual issue associated with this. I am moving this to a discussion.

To me this primarily raises the point that we need comprehensive documentation on what the ModelPolisher actually does.
